### ISSUE TITLE
GUI - is config file used?

### ISSUE THREAD

**Issue by rufik** [openconnect/openconnect-gui#116]
Is it possible to use config file (as openconnect.exe --config does) by GUI?
If not would it be added in future releases?

**Comment by horar** [comment]
Hi, there is no feature like this in GUI at this time. And I don't see benefits of it for GUI application. Yes, it make sense for console application.
Theoretically it would be nice to have a possibility to import/export particular profile so user may share them with colleagues ... but there is no request for this yet.

**Comment by rufik** [comment]
Using config file you can set up some option unavailable in GUI, ex. deflate, interface, no-xmlpost, verbosity.
For example, I have 2 TAP adapters - first for OpenVPN (client 1 network) and second for OpenConnect (client 2 network). I'd like to use "--interface" option to force OpenConnect to use second TAP adapter. Right now I have to edit vpn-script.js to get it working.

**Comment by horar** [comment]
Next options may be (and sooner-later will be) added into GUI.
But I think it's little bit useless to follow command line use case and start GUI application from console. This is already implemented in console client.
(e.g. the interface configuration selection is tracked here #103)

**Comment by rufik** [comment]
If you will add next options to GUI then nothing else is needed :)
First candidate is "--interface" option.

**Comment by mmercer-apixio** [comment]
Is there any status or update on this?  Our vpn requires the --no-xmlpost option, and unfortunately I am unable to pass this via the openconnect-gui package currently

**Comment by horar** [comment]
yes, next options are on the plan... but not early in Fall of this year
